guides/isolation.md

```diff
@@ -0,0 +1,178 @@
+# Instance and Database Isolation
+
+This guide will walk you through option for isolating Oban instances as well as Oban database
+tables.
+
+## Running Multiple Oban Instances
+
+You can run multiple Oban instances with different prefixes on the same system and have them
+entirely isolated, provided you give each Oban supervisor a distinct ID.
+
+Here we configure our application to start three Oban supervisors using the `"public"` (default),
+`"special"`, and `"private"` prefixes, respectively:
+
+```elixir
+def start(_type, _args) do
+  children = [
+    MyApp.Repo,
+    MyApp.Endpoint,
+    {Oban, name: ObanA, repo: MyApp.Repo},
+    {Oban, name: ObanB, repo: MyApp.Repo, prefix: "special"},
+    {Oban, name: ObanC, repo: MyApp.Repo, prefix: "private"}
+  ]
+
+  Supervisor.start_link(children, strategy: :one_for_one, name: MyApp.Supervisor)
+end
+```
+
+When you do this, you'll have to use the correct Oban supervisor name when performing Oban-related
```

Comment:
This would be a good place to mention facades, e.g. `use Oban`, which didn't exist when the isolation section was originally written. Or maybe it could go into a separate section in here?

Reply:
Yep, I can add it to a separate section.

Reply:
Done!